src/xpcc/architecture/platform/devices/stm32/stm32f072-c_r_v-8_b.xml

```diff
@@ -51,7 +51,7 @@
       <vector position="30" name="CEC_CAN"/>
       <vector position="31" name="USB"/>
     </driver>
-    <driver type="adc" name="stm32f0"/>
+<!--     <driver type="adc" name="stm32f0"/> -->
```

Comment:
But then wouldn't it be necessary for the driver to declare what devices it was implemented for? Otherwise that is an issue.